### DISCUSSION TITLE
Disable multiple companies

### DISCUSSION THREAD

**Discussion by dominikgeimer** [andrewdwallo/filament-companies#87] [General]
Hi,

is it possible to disable the creation of multiple companies? I just want to have one company after registration.

This seems not to work: 
https://github.com/andrewdwallo/filament-companies#switching-the-current-company

Thanks!

**Comment by andrewdwallo** [comment]
The mechanism for switching companies really has nothing to do with if a user is able to make more than one company. When switching companies, it updates the `current_company_id` in the database each time the switch is made. If you want to disable the creation of multiple companies, then you would just update the `create()` method in the Company Policy.

Here is an example:
```php
    /**
     * Determine whether the user can create models.
     */
    public function create(User $user): bool
    {
        return $user->ownedCompanies()->count() < 1;
    }
```

With this example, a user can't create more than one company.
